merlin/systems/triton/conversions.py

```diff
@@ -111,12 +111,19 @@ def _to_values_offsets(array):
     values, offsets
         Tuple of values and offsets
     """
-    num_rows = array.shape[0]
-    row_lengths = [array.shape[1]] * num_rows
-    offsets = [0] + list(itertools.accumulate(row_lengths))
     array_lib = cp if cp and isinstance(array, cp.ndarray) else np
+    shape = array.shape
+
+    values_shape = [-1]
+    if len(shape) > 2:
+        values_shape.extend(shape[2:])
+    values = array.reshape(*values_shape)
```

Comment:
since we only use values_shape in this call, are we setting it above to make this more readable?

Reply:
Yeah, it could be an if/else like this:
```python
if len(shape) > 2:
    array.reshape([-1, **shape[2:]])
else:
    array.reshape([-1])
```
That felt repetitive though so I figured I'd just do the reshape once.